core/domain/classifier/algorithm_registry_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for algorithm registry."""
+
+
+from core.domain.classifier import algorithm_registry
+from core.tests import test_utils
+from core.domain.classifier.LDAStringClassifier import LDAStringClassifier
+
+
+class AlgorithmRegistryTest(test_utils.GenericTestBase):
+    """Test the classifier algorithm registry class."""
+
+    def test_get_all_classifier_ids(self):
+        expected_algorithm_ids = [
+            'LDAStringClassifier'
+        ]
+        algorithm_ids = algorithm_registry.Registry.get_all_classifier_ids()
+        self.assertItemsEqual(expected_algorithm_ids, algorithm_ids)
+
+    def test_get_all_classifiers(self):
+        classifier_instances = algorithm_registry.Registry.get_all_classifiers()  # pylint: disable=line-too-long
+        self.assertEquals(len(classifier_instances), 1)
```

Comment:
Can we retrieve the expected results from `feconf`? 
This is a test case for `AlgorithmRegistry`, but every time we modify `feconf` it will be broke.